server/utils/utils.ts

```diff
@@ -11,8 +11,8 @@ import type { ServiceCatalogueService } from '../services'
 dayjs.extend(relativeTime.default)
 
 type HasName = { attributes?: { name: string } }
-type HasRepoName = { github_repo: string }
-type HasTeamName = { team_name: string }
+type HasRepoName = { github_repo?: string }
+type HasTeamName = { team_name?: string }
```

Comment:
There was something a bit weird going on here - these `github_repo` and `team_name` are optional on the underlying types but it wasn't complaining before.. now it correctly is so had to make these optional